### PR TITLE
Fix issue with parsing TRUNCATE statements, Issue #221

### PR DESCRIPTION
**Pull request by sinri**
Make the following SQL parsable, which is mentioned in issue #221 

`truncate table `a`.`b`;truncate c;select truncate from truncate;`

### CODE REVIEW

**Comment by sinri** [comment]
Wonder why conflicts came out.

**Comment by sinri** [comment]
Merged the latest master branch to resolve conflicts.

File: src/Token.php [hunk 1]
```diff
@@ -343,4 +343,14 @@ public function getInlineToken()
             $this->token
         );
     }
+
+    /**
+     * Make you happier when you need to debug.
```
Comment (ibennetch) [inline src/Token.php]: Is this function valuable to include in the code or is it debugging code that should be removed?

If it's worth keeping, I think the comment here should be changed to make the purpose of the function more clear.

**Comment by williamdes** [comment]
Merged as 04da59b51db477a9caaf3e755df414cf1b3c09c3
Thanks for your contribution and patience @sinri !

**Comment by sinri** [comment]
Great Result Finally!

**Pull request merged**